src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+function Books() {
+  return (
+    <div>
+      <h1>List of books:</h1>
+      <button type="button">Delete</button>
+      <br />
+      <br />
+      <br />
+      <p>Add New Book</p>
+      <form action="#">
+        <label htmlFor="name">
+          Book Name
+          <input type="text" name="name" id="name" />
+        </label>
+        <br />
+        <input type="submit" />
+      </form>
```

Comment:
- [ ] Nice Job but you missed one thing that the booklist should be in the component folder, as the duck patterns show.

![llll](https://user-images.githubusercontent.com/61562626/140207082-cb39befb-87c3-4e51-945e-eba3ea3d92f9.jpeg)

![ppppp](https://user-images.githubusercontent.com/61562626/140209877-0e9dc8ed-3ff0-4486-a930-51a8a1dc5d5e.jpeg)